src/useInfiniteScroll.js

```diff
@@ -40,12 +41,17 @@ function useInfiniteScroll({
     const rect = ref.current.getBoundingClientRect();
 
     const bottom = rect.bottom;
-    let bottomOffset = bottom - windowHeight;
+    let bottomOffset;
 
     if (scrollContainer === PARENT) {
       const { bottom: parentBottom } = getParentSizes();
       // Distance between bottom of list and its parent
       bottomOffset = bottom - parentBottom;
+    } else if (scrollContainer === SELF) {
+      const self = ref.current;
+      bottomOffset = self.scrollHeight - (self.scrollTop + self.clientHeight);
```

Comment:
When you try `scrollContainer: "self"` in the demo, it causes an infinite loop.
This line always returns `0`, and it constantly forces the hook to call `onLoadMore` prop.

Reply:
I will test again and fix it. Thanks